src/io/flutter/editor/ColorPickerFactory.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.editor;
+
+public class ColorPickerFactory {
+  static GenericColorPicker create() {
+    // TODO(jacobr): provide an implementation of GenericColorPicker that works
+    // in IntelliJ. com.intellij.ui.colorpicker.ColorPickerBuilder would work
+    // if we could figure out how to make it work with bin/plugin build.
```

Comment:
Yep, we modify the plugin tool to edit code during the build to handle compile problems. We could use git branches but for the small amount of changes we need to make that is more trouble than it's worth.

Reply:
had to expand the way it was modifying code a little to make it work as otherwise I get exceptions. see the stub placeholder file added.